.gitattributes

```diff
@@ -1,4 +1,6 @@
 * text=auto
 *.js text eol=lf
+*.log text eol=lf
+*.md.expected text eol=lf
```

Comment:
Btw, Git now supports setting `lf` for *all* text files (since 2016, but I didn't realize until recently):

```
* text=auto eol=lf
```

Reply:
Would that interfere with people's editor setup?